snuba/datasets/entities/discover.py

```diff
@@ -451,6 +452,15 @@ def selector_func(_query: Query) -> Tuple[str, List[str]]:
             ),
             join_relationships={},
             writable_storage=None,
+            required_conditions={
+                "project_id": [ConditionFunctions.EQ, ConditionFunctions.IN],
+                "timestamp": [
+                    ConditionFunctions.LT,
+                    ConditionFunctions.LTE,
+                    ConditionFunctions.GT,
+                    ConditionFunctions.GTE,
```

Comment:
Why not EQ or IN?

Reply:
Yeah I left those out because the extensions as they are now don't use those, but I'll add them in too.